addons/frag/functions/fnc_initBlackList.sqf

```diff
@@ -24,17 +24,16 @@ if (!ADDON) then {
 // could improve text parsing of CBA setting string
 private _convArray = parseSimpleArray GVAR(blackList);
 
-if (count _convArray == 0 ) exitWith {
+if (_convArray isEqualTo []) exitWith {
     TRACE_1("Empty blacklist", _convArray);
 };
 
 // Add CBA setting blacklist to blacklist and log errors
 private _errors = 0;
-private _items = count _convArray;
-for "_i" from 0 to _items - 1 do {
-    private _ammo = _convArray#_i;
-    if (typeName _ammo isNotEqualTo "STRING") then {
-        INFO_1("Improper ammo string at index %1", _i);
+{
+    private _ammo = _x;
+    if (!(_ammo isEqualType "")) then {
```

Comment:
Can actually do this in the future for less work:
```suggestion
    if !(_ammo isEqualType "") then {
```

Reply:
Isn't that against the ACE coding guidelines?

Reply:
Good point, I dunno.

Reply:
*technically*, yes

It's not as strictly enforced as brackets/tabs/params/variable names, though

Reply:
I think it should be allowed in cases like this where adding extra brackets looks really dumb

Reply:
Added in https://github.com/acemod/ACE3/pull/9728/commits/8dd76d1ba6c6658f0476b4ac76bcb6e2915d35c7